features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/vitals/JankStatsActivityLifecycleListener.kt

```diff
@@ -24,17 +32,45 @@ import java.util.concurrent.TimeUnit
 internal class JankStatsActivityLifecycleListener(
     private val vitalObserver: VitalObserver,
     private val internalLogger: InternalLogger,
-    private val jankStatsProvider: JankStatsProvider = JankStatsProvider.DEFAULT
+    private val jankStatsProvider: JankStatsProvider = JankStatsProvider.DEFAULT,
+    private var screenRefreshRate: Double = 60.0
 ) : ActivityLifecycleCallbacks, JankStats.OnFrameListener {
 
```

Comment:
We can remove it from the constructor, I don't see any call sites where we pass custom `screenRefreshRate` at the `JankStatsActivityLifecycleListener` construction time

```suggestion
) : ActivityLifecycleCallbacks, JankStats.OnFrameListener {
    private var screenRefreshRate: Double = 60.0
```

Reply:
It is used in the test ``𝕄 adjust sample value to refresh rate 𝕎 doFrame() {refresh rate over 60hz}`